workflow/scripts/build_scenarios.py

```diff
@@ -83,21 +88,26 @@ def write_to_scenario_yaml(output, scenarios, transport_share, naval_share):
         snakemake = mock_snakemake("build_scenarios")
 
     # Set USERNAME and PASSWORD for the Ariadne DB
-    pyam.iiasa.set_config(
-        snakemake.params.iiasa_usr, 
-        snakemake.params.iiasa_pwd,
+    ariadne = pd.read_csv(
+        snakemake.input.ariadne_database,
+        index_col=["model", "scenario", "region", "variable", "unit"]
     )
+    ariadne.columns = ariadne.columns.astype(int)
 
-    model_df= pyam.read_iiasa(
-        "ariadne_intern",
-        model=snakemake.params.iiasa_model,
-        scenario=snakemake.params.iiasa_scenario,
-    ).timeseries()
+    df = ariadne.loc[
+        :, 
+        snakemake.params.iiasa_scenario, 
+        "Deutschland"]
 
-    df = model_df.loc[snakemake.params.iiasa_model, snakemake.params.iiasa_scenario, "Deutschland"]
 
-    transport_share, naval_share = get_shares(df)
+    planning_horizons = [2020, 2025, 2030, 2035, 2040, 2045]
+    transport_share, naval_share = get_shares(df, planning_horizons)
 
     scenarios = snakemake.params.scenario_name
-    output = snakemake.input[0]
-    write_to_scenario_yaml(output, scenarios, transport_share, naval_share)
\ No newline at end of file
+
+    if "snakemake" in globals():
```

Comment:
Why? with mocksnakemake I find the same path with `snakemake.input.scenario_yaml`